src/variorum/variorum.h

```diff
@@ -149,6 +142,18 @@ int variorum_cap_each_core_frequency_limit(int cpu_freq_mhz);
 /// not supported, otherwise -1
 int variorum_cap_socket_frequency_limit(int socketid, int socket_freq_mhz);
 
+/// @brief Cap the power usage identically of each GPU on the node.
+///
+/// @supparch
+/// - NVIDIA Volta and later
+/// - WIP: AMD
+///
+/// @param [in] gpu_power_limit Desired power limit in watts for each GPU
+///             on the node.
+///
+/// @return 0 if successful, otherwise -1
+int variorum_cap_gpu_power_limit(int gpu_power_limit);
```

Comment:
Do we want to call this `variorum_cap_gpu_power_limit` or `variorum_cap_each_gpu_power_limit`? Our convention so far was to use `each` to indicate a uniform power cap on APIs where we don't specify the device ID. (See the socket power cap or frequency cap examples). We are going to redesign the power capping API soon, so not sure if this matters. But in your PR, you deleted the previously commented version of `cap_each_gpu_power_limit` and dropped the `each` -- not sure if we want to keep it. Will defer to Stephanie here.  In general, in the future, we want to say something clearer in the APIs: eg `cap_uniform_power_limit(device type, device ID, power limit)` for example (divide type can be socket, core, Memory, GPU, device ID is the numeric ID of the device, eg the 10th core or 3rd GPU, and the limit is self-explanatory).

Reply:
I think we want to use `each` if we've been using that in existing convention. Then we can make the change to something more clear and specific across the board at once.

Reply:
Agreed, so then in this PR, we need to change `variorum_cap_gpu_power_limit` --> `variorum_cap_each_gpu_power_limit` everywhere. Tagging @amarathe84 so he gets notified.

Reply:
Apparently I need to leave a comment for me to indicate "changes requested" in the review, so here it is :D

Reply:
I think I was following the API naming convention for GPU power capping ratio and socket frequency limit. Both APIs don't have 'each' so I skipped it for the GPU capping API as well. I agree with the suggested change and I'm changing the API name to `variorum_cap_each_gpu_power_limit` as it makes more sense (also updating the docs). We'll streamline the other two (ratio, frequency) in a separate PR for API rehaul work.